src/mobile/package.json

```diff
@@ -10,11 +10,16 @@
     "eject": "expo eject"
   },
   "dependencies": {
+    "@react-navigation/bottom-tabs": "^6.1.0",
+    "@react-navigation/native": "^6.0.7",
+    "@react-navigation/native-stack": "^6.3.0",
     "expo": "44.0.5",
     "expo-status-bar": "1.2.0",
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-native": "0.67.1",
+    "react-native-safe-area-context": "3.3.2",
+    "react-native-screens": "~3.10.1",
```

Comment:
Great to see telescope on mobile

Do we want to pin deps when installing or let Renovate do it (Renovate is broken atm)?

Reply:
@TueeNguyen I just start knwoing how to `pin` deps and I know we do that because we want to have a stable deployment of our project but what if we have security error in our deps ?

Reply:
Hi, we need to pin these dependencies because they're working fine now but in the future they may break our build. `^6.3.0` means that whenever you do `pnpm install`, it'll install a package with version `> 6.3.0  && < 7.0.0`. `6.4.0` version may contain code that doesn't work. 

It's been more alarming lately https://www.youtube.com/watch?v=R6S-b_k-ZKY